cilium/cmd/kernel_check.go

```diff
@@ -41,6 +41,10 @@ type KernelParamSupport struct {
 
 func kernelCheck() {
 	probeManager := probes.NewProbeManager()
+	if probeManager.KernelConfigAvailable() != true {
+		Fatalf("Unable to detect BPF support. Is the kernel config available?")
```

Comment:
```suggestion
		Fatalf("Could not find kernel config file")
```
Or something like this, since we already know we couldn't find kernel config.